common/xrdp_client_info.h

```diff
@@ -143,7 +143,7 @@ struct xrdp_client_info
   int use_frame_acks;
   int max_unacknowledged_frame_count;
 
-  int disableSSLv3; /* 0 = no, 1 = yes */
+  long ssl_protocols;
```

Comment:
The line would be between `options` (the last field used by xorgxrdp) and `h264_codec_id` (any fields below are not used by X11rdp or xorgxrdp). `disableSSLv3` is in the "safe" zone. We still need some way to pass newly added messages to the backends.